www/md_base/src/app/home/home.route.coffee

```diff
@@ -17,3 +17,33 @@ class State extends Config
             templateUrl: "views/#{name}.html"
             name: name
             url: '/'
+
+        bbSettingsServiceProvider.addSettingsGroup
+            name: 'home'
+            caption: 'Home'
+            description: 'Settings to customize the display of Home page'
+            items: [
+                type: 'bool'
+                name: 'lock_panels'
+                caption: 'Lock panels'
+                default_value: false
+            ,
+                type: 'hidden'
+                name: 'panels'
+                default_value: [
+                    name: 'overview'
+                    title: 'overview'
+                    collapsed: false
+                    template: 'views/overview_panel.html'
```

Comment:
On the contrary, this is making a lot of sense for me, and is very much in the spirit of angularJS 2.0, where everything is a directive.